apiary.apib

```diff
@@ -0,0 +1,515 @@
+FORMAT: 1A
+HOST: https://polls.apiblueprint.org/
+
+# IFN711
+
+Polls is a simple API allowing consumers to view polls and vote in them.
+
+
+
+## Suppliers [/suppliers]
+
+### Current Suppliers [GET]
+
++ Response 200 (application/json)
+
+                [
+                    {
+                        "supplierID": 1,
+                        "supplier": [
```

Comment:
Supplier and remove the brackets "[", this represents an array, we are the data represent only one Supplier.